e2e/LongHaul/service/Program.cs

```diff
@@ -102,6 +104,11 @@ await Task
             s_aiLoggingProvider.Dispose();
         }
 
+        private static int PortFilter(Parameters parameters)
+        {
+            return parameters.TransportProtocol == IotHubTransportProtocol.WebSocket ? 443 : 5671;
```

Comment:
The problem with this is that even if we choose to use TCP or WebSocket here, AMQP operations on the client will still be on a different port.

Reply:
@drwill-ms should I remove this implementation or you think there might be a workaround here?

Reply:
Can you filter in more than 1 port?

Reply:
sure, we can pass in array of ports as a parameter when filtering.

Reply:
Then for the device client, always include 443 for HTTPS, and for the service client, always include 443 for HTTP and 5671 for AMQP.